google/cloud/bigquery/v2/minimal/internal/job_options_test.cc

```diff
@@ -48,6 +48,9 @@ TEST(JobOptionstTest, DefaultOptions) {
             expected_retry->IsExhausted());
 
   EXPECT_TRUE(actual.has<BigQueryJobBackoffPolicyOption>());
+
+  EXPECT_TRUE(actual.has<BigQueryJobConnectionPoolSizeOption>());
+  EXPECT_GT(actual.get<BigQueryJobConnectionPoolSizeOption>(), 0);
```

Comment:
Aside: Note that if the first expectation fails, the second will too (because `std::size_t()` is 0).  We usually try to avoid such cascading errors, but not a big deal.